src/Microsoft.ApplicationInsights/Extensibility/ICloneable.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility
+{
+    /// <summary>
+    /// Interface for defining method to Deeply clone the members.
+    /// </summary>
+    public interface ICloneable
```

Comment:
The only consumer of this interface is IExtension. Why we need it as an interface then? Just define a method on that interface

Reply:
Right, removed.